.github/checks/check-prefix-overlap.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python3
+"""
+Check for overlapping IPv4/IPv6 prefixes across all location files and
+ensure all prefixes are inside the allowed Freifunk Berlin ranges.
+
+Exit codes:
+ 0 - OK
+ 1 - any issue found (overlap or out-of-range)
+"""
+import glob
+import ipaddress
+import sys
+
+import yaml
+
+# Allowed Freifunk Berlin ip ranges
+ALLOWED_IPV4 = [
+    ipaddress.ip_network("10.31.0.0/16"),
+    ipaddress.ip_network("10.36.0.0/16"),
+    ipaddress.ip_network("10.230.0.0/16"),
+    ipaddress.ip_network("10.248.0.0/14"),
+]
+
+ALLOWED_IPV6 = [
+    ipaddress.ip_network("2001:bf7:750::/44"),
+    ipaddress.ip_network("2001:bf7:760::/44"),
+    ipaddress.ip_network("2001:bf7:770::/44"),
+    ipaddress.ip_network("2001:bf7:780::/44"),
+    ipaddress.ip_network("2001:bf7:790::/44"),
+    ipaddress.ip_network("2001:bf7:800::/44"),
+    ipaddress.ip_network("2001:bf7:810::/44"),
+    ipaddress.ip_network("2001:bf7:820::/44"),
+    ipaddress.ip_network("2001:bf7:830::/44"),
+    ipaddress.ip_network("2001:bf7:840::/44"),
+    ipaddress.ip_network("2001:bf7:850::/44"),
+    ipaddress.ip_network("2001:bf7:860::/44"),
+]
+
+
+def load_prefixes(paths):
+    prefixes = []
+    for p in paths:
+        try:
+            with open(p) as f:
+                doc = yaml.safe_load(f) or {}
+        except Exception:
+            continue
```

Comment:
probably here using try catch without handling exception can be discussed, however, its fine for me :)